spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -38,6 +54,113 @@
           expect(cur_plat._platform.target_user).to eq(plat[:target_user])
         end
       end
+
+      describe '#generate_msi_packaging_artifacts' do
+        before(:each) do
```

Comment:
@haus I don't see what the difference would be, and we are actually doing setup and tear down anyway so to me it would make more sense with before and after.